editor/plugins/audio_stream_editor_plugin.cpp

```diff
@@ -0,0 +1,284 @@
+/*************************************************************************/
+/*  audio_editor_plugin.cpp                                              */
```

Comment:
Mismatched comment and the file name, also in .h file.

Reply:
ouch...